components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
Note that people who use gateway only will be forced to rerender thumbnails? I personally wouldn't touch it!